company-profiles/memory.md

```diff
@@ -0,0 +1,30 @@
+# Memory
+
+## Company blurb
+
+Memory is not your average Norwegian company. We work tirelessly because we have a need build something bigger than ourselves. To create.
+
+## Company size
+
+We’re a team of 45 people.
+
+## Remote status
+
+Memory is global company, but we have our roots in Norway. Our team is diverse and represents 20 countries. We’re also big believers in remote work.
```

Comment:
Typo: "Memory is **a** global company"

Reply:
Fixed in https://github.com/remoteintech/remote-jobs/pull/618/commits/cc608f585280e66454b2039a1555db4e8cbe4df0